superset/models/core.py

```diff
@@ -249,6 +250,14 @@ def parameters(self) -> Dict[str, Any]:
 
         return parameters
 
+    @property
+    def parameters_schema(self) -> Dict[str, Any]:
+        try:
+            parameters_schema = self.db_engine_spec.parameters_json_schema()  # type: ignore # pylint: disable=line-too-long,useless-suppression
```

Comment:
This line can be cleaned up a little bit. If you're disabling the `useless-suppression` lint rule it means there's another rule you're suppressing unnecessarily. You can probably write this as (untested):

```suggestion
            parameters_schema = self.db_engine_spec.parameters_json_schema()  # type: ignore
```

Reply:
when I had this with just type: ignore it was failing the python lint test. So I emulated this line:

https://github.com/apache/superset/blob/master/superset/models/core.py#L246